src/golang/cmd/migrator/versions/000019_add_serialization_value_to_param_op/utils.go

```diff
@@ -0,0 +1,110 @@
+package _000019_add_serialization_value_to_param_op
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/utils"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	serilization_type_key   = "serialization_type"
+	value_key               = "val"
+	spec_field              = "spec"
+	pythonExecutorPackage   = "aqueduct_executor"
+	typeInferencePythonPath = "migrators.parameter_val_type_inference_000019.main"
+)
+
+type Operator struct {
+	Id     uuid.UUID `db:"id" json:"id"`
+	OpSpec Spec      `db:"spec" json:"spec"`
+}
+
+func getAllOperators(
+	ctx context.Context,
+	db database.Database,
+) ([]Operator, error) {
+	query := "SELECT id, spec FROM operator;"
+
+	var response []Operator
+	err := db.Query(ctx, &response, query)
+	return response, err
+}
+
+func updateParamOperatorWithNewSpec(
+	ctx context.Context,
+	operator Operator,
+	db database.Database,
+) error {
+	param_val := operator.OpSpec.Param[value_key]
+	// Launch the Python job to infer the type of the parameter value
+	cmd := exec.Command(
+		"python3",
+		"-m",
+		fmt.Sprintf("%s.%s", pythonExecutorPackage, typeInferencePythonPath),
+		"--val",
+		param_val,
```

Comment:
I assume this is fine because we are assuming previous parameter values won't be very big, so passing it directly as cmd will work?

Reply:
yes, the val was passed as part of the spec and had to be accepted via cmd so it should preserve the property

Reply:
I think that's a reasonable assumption if we don't expect to ever downgrade and then re-upgrade this version right?
Otherwise, I wonder if it'll hang on the re-upgrade part, if they had large parameters in the newer version.

Reply:
Nah let's not worry about the re-upgrade part. :)